lms/djangoapps/course_home_api/outline/v1/serializers.py

```diff
@@ -21,8 +21,35 @@ def get_url(self, tool):
         return request.build_absolute_uri(url)
 
 
+class CourseBlockSerializer(serializers.Serializer):
+    """
+    Serializer for Course Block Objects
+    """
+    blocks = serializers.SerializerMethodField()
+
+    def get_blocks(self, blocks):
+        data = {}
+
+        for block_key in blocks:
+            block_data = {
+                'id': str(block_key),
+                'type': str(block_key.category),
```

Comment:
Nit: Does category need to be `str` or is it a string already? I would think it is a string